ckanext/ldap/plugin.py

```diff
@@ -74,7 +74,9 @@ def configure(self, main_config):
             'ckanext.ldap.organization.role': {'default': 'member', 'validate': _allowed_roles},
             'ckanext.ldap.ckan_fallback': {'default': False, 'parse': p.toolkit.asbool},
             'ckanext.ldap.prevent_edits': {'default': False, 'parse': p.toolkit.asbool},
-            'ckanext.ldap.migrate': {'default': False, 'parse': p.toolkit.asbool}
+            'ckanext.ldap.migrate': {'default': False, 'parse': p.toolkit.asbool},
+            'ckanext.ldap.debug_level': {'default': '0'},
+            'ckanext.ldap.trace_level': {'default': '0'}
```

Comment:
These could be defined as `{'default': 0, 'parse': p.toolkit.asint}` then you don't have to check the value in the code, you can just use it.